.gitignore

```diff
@@ -21,3 +21,4 @@ yarn-error.log*
 
 #TODO add .env to your gitignore
 
+server/.env
```

Comment:
You will need to `git rm ` this file as well, otherwise it will still be tracked